SharpPcap/LibPcap/PcapInterface.cs

```diff
@@ -74,53 +74,12 @@ public class PcapInterface
         /// </value>
```

Comment:
can you split this commit up between the cleanup changes (which look good) and the changes to fix the null exception (which also look good)? Otherwise its a challenge to separate out the fix null exception lines from the not necessarily required, but good looking, cleanups.

Reply:
The cleanup in `SharpPcap/LibPcap/Sockaddr.cs` could be moved to its own PR.
There isn't much cleanup in `SharpPcap/LibPcap/PcapInterface.cs` to deserve its own PR

Reply:
@kayoub5 same PR works if you can put them into a separate commit.